learn/cloud/analytics.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: Analytics
+description: Meilisearch Cloud offers in-depth search analytics to help you understand how users search in your application.
+sidebarDepth: 3
+---
+
+# Analytics
+
+Enable Meilisearch Cloud analytics to gain insight on what your users are searching for, how often they find it, and how much they have to wait between typing a query and receiving results.
+
+## Requirements
+
+You must have a Meilisearch Cloud account to have access to search analytics.
```

Comment:
The requirements seem like the appropriate section to create an account if you don't have one.

I would add a link to `https://meilisearch.com/cloud?utm_campaign=oss&utm_source=docs&utm_medium=analytics` here.